### DISCUSSION TITLE
Buttons Deactivated

### DISCUSSION THREAD

**Discussion by bhavya-error404** [github/india#289] [Students]
I just completed the orientation video and updated my LinkedIn with Microsoft Azure skill, but now all the three tabs i.e. Self Learning, Industry and Project are deactivated ( I can not click onto them earlier I was able to). Please let me know what am I missing here.

**Comment by scmishra-cse** [comment]
Hi @bhavya-error404 The same thing happened with me Yesterday night, but now everything is fine, I'll suggest you to use Microsoft edge browser for all future ready talent stuffs something chrome browser makes it complicated. Try this then tell me your problem is resolved or not still if you are having any issues just let me know!

**Reply by bhavya-error404** [reply]
Still not working. I have to inspect the element, changing the HTML attributes and then I am accessing the courses but that is not the solution. One question the orange progress bar is it midway or completed in yours?

**Reply by TheSuranaverse** [reply]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
